src/Components/NotFoundErrorPage.tsx

```diff
@@ -43,3 +60,27 @@ export const NotFoundErrorPage = connect(function NotFoundErrorPage() {
     </>
   )
 })
+
+const NoRootError = connect(
```

Comment:
* [ ] FYI to be more inline with the naming of the other components in this module: `NoRoot`.